extensions/shared/blocks/cover/cover-media-placeholder.js

```diff
@@ -19,23 +18,23 @@ const ALLOWED_MEDIA_TYPES = [ 'image', 'video' ];
 export default createHigherOrderComponent(
 	CoreMediaPlaceholder => props => {
 		const { name } = useBlockEditContext();
-		if ( ! isUpgradable( name ) ) {
+		if ( 'core/cover' !== name ) {
 			return <CoreMediaPlaceholder { ...props } />;
 		}
 
 		const { onError } = props;
 		return (
 			<div className="jetpack-cover-media-placeholder">
 				<CoverMediaContext.Consumer>
-					{ ( onFilesUpload ) => (
+					{ onFilesUpload => (
```

Comment:
This and the next two changes seem to have been automated?

Reply:
yes, I think so